Public/Src/IDE/VsCode/updateVsixTemplate.cmd

```diff
@@ -1,23 +0,0 @@
-@echo off

```

Comment:
can't we now remove everything under pluginTemplate as well?

Reply:
yes, we can now delete `pluginTemplate/extension`.  Doing that would make this PR have over a 1000 modified files though.  So after this PR gets approved, we can delete that folder.

Reply:
We will delete this folder in a separate pull request